src/applications/covid19-chatbot/index.js

```diff
@@ -0,0 +1,170 @@
+import 'botframework-webchat';
+
+const defaultLocale = 'en-US';
+const localeRegExPattern = /^[a-z]{2}(-[A-Z]{2})?$/;
+let chatBotScenario = 'unknown';
+let root = null;
+
+function extractLocale(localeParam) {
+  if (localeParam === 'autodetect') {
+    return navigator.language;
+  }
+
+  // Before assigning, ensure it's a valid locale string (xx or xx-XX)
+  if (localeParam.search(localeRegExPattern) === 0) {
+    return localeParam;
+  }
+  return defaultLocale;
+}
+
+function getUserLocation(callback) {
+  navigator.geolocation.getCurrentPosition(
+    position => {
+      const latitude = position.coords.latitude;
+      const longitude = position.coords.longitude;
+      const location = {
+        lat: latitude,
+        long: longitude,
+      };
+      callback(location);
+    },
+    error => {
+      // user declined to share location
+      // eslint-disable-next-line no-console
+      console.log(`location error:${error.message}`);
+      callback();
+    },
+  );
+}
+
+function startChat(user, webchatOptions) {
+  window.WebChat.renderWebChat(webchatOptions, root);
+}
+
+function initBotConversation() {
+  if (this.status >= 400) {
+    alert(this.statusText);
+    return;
+  }
+  // extract the data from the JWT
+  const jsonWebToken = this.response;
```

Comment:
the api is returning json, will this work?

Reply:
what i mean is, the previous code expected that the JWT would be returned as the full response. Now it is returning as such:
{ token: (the token) }

@shaunhubbard-va can clarify here, but will the above code work in that case?

Reply:
@shaunhubbard-va can you add additional details here?